src/elements/tabs/tab/tab.ts

```diff
@@ -0,0 +1,51 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import type { SbbTabLabelElement } from '../tab-label.js';
+
+import style from './tab.scss?lit&inline';
+
+/**
+ * Combined with a `sbb-tab-group`, it displays a tab's content.
+ *
+ * @slot - Use the unnamed slot to provide content.
+ */
+@customElement('sbb-tab')
+export class SbbTabElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /** The `sbb-tab-label` associated with the tab. */
+  public get label(): SbbTabLabelElement | null {
+    return this._label;
+  }
+  private _label: SbbTabLabelElement | null = null;
+
+  private _getTabLabel(): SbbTabLabelElement | null {
+    let previousSibling = this.previousElementSibling;
+    while (previousSibling && previousSibling.localName !== 'sbb-tab-label') {
+      previousSibling = previousSibling.previousElementSibling;
+    }
+    return previousSibling as SbbTabLabelElement;
+  }
+
+  /** @internal */
+  public configure(): void {
```

Comment:
Maybe call the method
```suggestion
  public syncLabel(): void {
```
?

More generally I'm not sure why the label is private and has a getter and additionally will not be used by this component at all. So my question - is it the best place to have all the label logic? Or should it completely be removed from the tab?

Reply:
This was done looking at the stepper as requested in #2240 ; for now I think it's almost useless, but could be a starting point for the next migration/refactoring, in which the whole 'configure' logic (select, deselect..) should be moved from the tab-group to the tab/tab-label, as done in the stepper.

IMO we can remove this logic at all and add it again later.

Reply:
It's difficult to say. I think it's ok for now. the main goal is that the public API has its breaking change and internal stuff we could change at a later point